README.md

```
@@ -1,3 +1,29 @@
-# java-janggi
+# **java-janggi**
 
 장기 미션 저장소
+
+### **기능 요구사항**
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)